### PR TITLE
Updatelogin

### PR DESCRIPTION
**Pull request by elharo**
currently failing with

[ERROR]   Missing requirement: com.google.cloud.tools.eclipse.3rdparty.feature.feature.group 0.1.0.qualifier requires 'org.eclipse.equinox.p2.iu; com.google.api.services.oauth2 [1.25.0,1.25.1)' but it could not be found

May need to upgrade all of those first, or in sync with this

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -95,6 +95,18 @@
...
     </dependency>
+    <dependency>
+      <!-- bundle: com.google.cloud.tools.login.plugins-login-common -->
+      <groupId>com.google.cloud.tools.login</groupId>
+      <artifactId>plugins-login-common</artifactId>
+      <version>0.2.2</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.apis</groupId>
+      <artifactId>google-api-services-oauth2</artifactId>
```
Comment (chanseokoh) [inline pom.xml]: I see this one is not OSGified (no OSGi info in the JAR metadata), so yeah, this won't work, and it explains the error, right?

Comment (elharo) [reply pom.xml]: Makes sense. Let me get on that. We may need to do this in a different order of fixes.

Comment (elharo) [reply pom.xml]: b/116784645

**Comment by elharo** [comment]
[INFO] The following dependencies in Dependencies have newer versions:
[INFO]   com.google.api-client:google-api-client ............. 1.23.0 -> 1.25.0
[INFO]   com.google.apis:google-api-services-appengine ...
[INFO]                                       v1-rev35-1.23.0 -> v1-rev80-1.25.0
[INFO]   com.google.apis:google-api-services-bigquery ...
[INFO]                                     v2-rev362-1.23.0 -> v2-rev405-1.25.0
[INFO]   com.google.apis:google-api-services-cloudresourcemanager ...
[INFO]                                      v1-rev455-1.23.0 -> v2-rev34-1.25.0
[INFO]   com.google.apis:google-api-services-compute ...
[INFO]                                     v1-rev160-1.23.0 -> v1-rev198-1.25.0
[INFO]   com.google.apis:google-api-services-dataflow ...
[INFO]                                   v1b3-rev219-1.23.0 -> v1b4-rev7-1.20.0
[INFO]   com.google.apis:google-api-services-datastore ...
[INFO]                                       v1-rev34-1.23.0 -> v1-rev63-1.25.0
[INFO]   com.google.apis:google-api-services-iam ...
[INFO]                                     v1-rev222-1.23.0 -> v1-rev261-1.25.0
[INFO]   com.google.apis:google-api-services-oauth2 ...
[INFO]                                     v2-rev131-1.23.0 -> v2-rev141-1.25.0
[INFO]   com.google.apis:google-api-services-pubsub ...
[INFO]                                     v1-rev363-1.23.0 -> v1-rev403-1.25.0
[INFO]   com.google.apis:google-api-services-servicemanagement ...
[INFO]                                     v1-rev368-1.23.0 -> v1-rev412-1.25.0
[INFO]   com.google.apis:google-api-services-storage ...
[INFO]                                     v1-rev115-1.23.0 -> v1-rev138-1.25.0
[INFO]   com.google.cloud.tools.login:plugins-login-common ..... 0.1.0 -> 0.2.2
[INFO]   com.google.http-client:google-http-client ........... 1.23.0 -> 1.25.0
[INFO]   com.google.http-client:google-http-client-jackson ... 1.23.0 -> 1.25.0
[INFO]   com.google.http-client:google-http-client-jackson2 ...
[INFO]                                                         1.23.0 -> 1.25.0
[INFO]   com.google.oauth-client:google-oauth-client ......... 1.23.0 -> 1.25.0
[INFO]   com.google.oauth-client:google-oauth-client-java6 ... 1.23.0 -> 1.25.0
[INFO]   com.google.oauth-client:google-oauth-client-jetty ... 1.23.0 -> 1.25.0
[INFO]

**Comment by elharo** [comment]
Build failure is weird:


18538 [ERROR] Failed to resolve target definition /home/travis/build/GoogleCloudPlatform/google-cloud-eclipse/eclipse/oxygen/gcp-eclipse-oxygen.target: Could not find "org.eclipse.linuxtools.docker.feature.feature.group/4.0.1.201809121533" in the repositories of the current location -> [Help 1]

**Comment by chanseokoh** [comment]
#3255 happening again, locally too. Maybe we have to do the same fix again. @briandealwis why is linuxtools breaking so frequently? Is it like they deprecate old repositories quickly?

**Comment by briandealwis** [comment]
Oh.  Their `update-docker` must be pointing to the very latest build.

**Pull request closed without merging**